seed/static/seed/partials/inventory_detail.html

```diff
@@ -20,8 +20,13 @@ <h1>{$:: (inventory_type === 'taxlots' ? 'Tax Lot Detail' : 'Property Detail') |
             <div class="section_action_container left" style="width: 50%;">
                 <span>
                     <h2>
-                        <span ng-if="::inventory_type==='properties'"><i class="fa fa-building-o"></i><span translate>Property</span> : {$ item_state.address_line_1 ? item_state.address_line_1 : ('(no address 1)' | translate) $}</span>
-                        <span ng-if="::inventory_type==='taxlots'"><i class="fa fa-map-o"></i><span translate>Tax Lot</span> : {$ item_state.address_line_1 ? item_state.address_line_1 : ('(no address 1)' | translate) $}</span>
+                        <span ng-if="::inventory_type==='properties'">
+                            <i class="fa fa-building-o"></i><span translate>Property</span> : 
+                        </span>
+                        <span ng-if="::inventory_type==='taxlots'">
+                            <i class="fa fa-map-o"></i><span translate>Tax Lot</span> : 
+                        </span>
+                        <span ng-bind-html="inventory_name">{$ inventory_display_name(inventory_type==='properties' ? 'property' : 'taxlot') | translate $}</span>
```

Comment:
Looks like sanitization is working properly 😉 
<img width="879" alt="Screen Shot 2021-01-29 at 2 21 07 PM" src="https://user-images.githubusercontent.com/18518728/106328748-965aeb00-623d-11eb-815e-863304d8c2ae.png">
<img width="423" alt="Screen Shot 2021-01-29 at 2 23 13 PM" src="https://user-images.githubusercontent.com/18518728/106328757-9955db80-623d-11eb-96d7-6d604dacbc7d.png">